Dockerfile

```diff
@@ -28,12 +29,15 @@ RUN apt-get update -qq \
     && mkdir -p /var/www/vhosts/${VHOST}/private \
     && mkdir -p /var/www/vhosts/${VHOST}/cgi-bin \
     && mkdir -p /var/www/vhosts/${VHOST}/htdocs \
-    && mkdir -p /var/www/vhosts/${VHOST}/logs \
-    && pip3 install -q --upgrade pip==21.2.4 setuptools --index-url ${PYPI_URL}
+    && mkdir -p /var/www/vhosts/${VHOST}/logs
+
+RUN pip3 install pipenv && pipenv --version
```

Comment:
Installation of pipenv should stay in the same docker layer as the other apt install.